draft-ietf-tls-sni-encryption.md

```diff
@@ -109,13 +138,14 @@ message.
 The SNI was defined to facilitate management of servers, but the
 developers of middleboxes found out that they could take
 advantage of the information. Many examples of such usage are
-reviewed in [@?RFC8404]. They include:
+reviewed in [@?RFC8404], other examples came out during discussions
```

Comment:
nit: this is a comma splice.

Reply:
I think that's fixed now, using a period instead of a comma.